### PR TITLE
[TD-11064]<test>: add schemaless function test and benchmark

### PR DESCRIPTION
**Pull request by huskar-t**
@zitsen @sangshuduo

### CODE REVIEW

File: wrapper/schemaless_test.go [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+	for i := 0; i < b.N; i++ {
+		result := wrapper.TaosSchemalessInsert(conn, []string{
+			"sys_if_bytes_out 1636626444 1.3E3 host=web01 interface=eth0",
+		}, wrapper.OpenTSDBTelnetLineProtocol, "")
+		code := wrapper.TaosError(result)
+		if code != 0 {
+			errStr := wrapper.TaosErrorStr(result)
+			wrapper.TaosFreeResult(result)
+			b.Error(&errors.TaosError{
+				Code:   int32(code) & 0xffff,
```
Comment (sangshuduo) [inline wrapper/schemaless_test.go:35]: does the mask potentially make the error code be mistake read (in future??

File: wrapper/schemaless_test.go [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+func TestSchemalessTelnet(t *testing.T) {
+	result := wrapper.TaosSchemalessInsert(conn, []string{
+		"sys_if_bytes_out 1636626444 1.3E3 host=web01 interface=eth0",
+	}, wrapper.OpenTSDBTelnetLineProtocol, "")
+	code := wrapper.TaosError(result)
+	if code != 0 {
+		errStr := wrapper.TaosErrorStr(result)
+		wrapper.TaosFreeResult(result)
+		t.Error(&errors.TaosError{
+			Code:   int32(code) & 0xffff,
```
Comment (sangshuduo) [inline wrapper/schemaless_test.go:54]: same question

File: wrapper/schemaless_test.go [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+	s := time.Now()
+	result = wrapper.TaosSchemalessInsert(conn, []string{
+		"sys_if_bytes_out 1636626444 1.3E3 host=web01 interface=eth0",
+	}, wrapper.OpenTSDBTelnetLineProtocol, "")
+	code = wrapper.TaosError(result)
+	if code != 0 {
+		errStr := wrapper.TaosErrorStr(result)
+		wrapper.TaosFreeResult(result)
+		t.Error(&errors.TaosError{
+			Code:   int32(code) & 0xffff,
```
Comment (sangshuduo) [inline wrapper/schemaless_test.go:69]: same question

File: wrapper/schemaless_test.go (hunk 1 above)
Comment (zitsen) [reply wrapper/schemaless_test.go]: I've added NewError function in errors package and updated test code.

**Review by zitsen — APPROVED** [review]
LGTM

**Review by sangshuduo — APPROVED** [review]
LGTM

**Pull request merged**